src/components/Envelope.vue

```diff
@@ -249,7 +249,7 @@ export default {
 			return this.data.flags.draft
 		},
 		account() {
-			const accountId = this.data.accountId ?? this.data.accountId
```

Comment:
I don't think the accountId is missing still on envelopes. That was only the case for messages loaded in a paged result. After https://github.com/nextcloud/mail/pull/5601 this shouldn't happen anymore

Reply:
that is true. it should just be `const accountId = this.data.accountId` :)